src/Form/DCBComponentRegionForm.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Drupal\dcb\Form;
+
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\dcb\Controller\DCBRegionController;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Cache\CacheTagsInvalidator;
+
+
+/**
+ * Class DCBComponentRegionForm
+ *
+ * @package Drupal\dcb\Form
+ */
+class DCBComponentRegionForm extends FormBase {
+
+  /**
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManager;
+
+  /**
+   * @var \Drupal\Core\Cache\CacheTagsInvalidator
+   */
+  protected $cacheTagsInvalidator;
+
+  /**
+   * DCBComponentRegionForm constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManager      $entityTypeManager
+   * @param \Drupal\Core\Cache\CacheTagsInvalidator    $cacheTagsInvalidator
+   * @param \Drupal\dcb\Controller\DCBRegionController $regionController
+   */
+  public function __construct(EntityTypeManager $entityTypeManager,
+                                 CacheTagsInvalidator $cacheTagsInvalidator,
+                                 DCBRegionController $regionController) {
+    $this->cacheTagsInvalidator = $cacheTagsInvalidator;
+    $this->entityTypeManager = $entityTypeManager;
+    $this->regionController = $regionController;
+  }
+
+  /**
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *
+   * @return static
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager'),
+      $container->get('cache_tags.invalidator'),
+      $container->get('dcb.region.controller')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'dcb_component_region_order';
+  }
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state,
```

Comment:
Two args on one line one dangling? I didn't realize I had OCD, you just proved I do.